ceph.spec.in

```diff
@@ -586,6 +573,40 @@ Requires:       python2-protobuf
 ceph-mgr-diskprediction-cloud is a ceph-mgr plugin that tries to predict
 disk failures using services in the Google cloud.
 
+%package mgr-essentials
```

Comment:
```suggestion
%package mgr-modules
```
? `essentials` sounds a bit misleading to me.

Reply:
IIUC this PR is splitting all the "essential" modules into a separate package, not **all** the modules. Maybe "mgr-essential-modules"?

Reply:
Another idea: `mgr-core-modules`

Reply:
i like core-modules better. actually i was using `ceph-mgr-modules-core`. but since i was worried that some of us might argue that it's inconsistent with other modules. as they are named like "ceph-mgr-diskprediction-local" without "module" in their names.

Reply:
`ceph-mgr-modules-core` sounds great to me!

Reply:
I do think we should avoid the word "plugin" because it's not obvious that "plugin" and "module" are the same thing.

Reply:
yeah, i keep forgetting which one we should be using. but after searching for plugin and modules in our document, i can find both of them! i am asking help in `#ceph-dashboard` for more inputs. IIRC, @LenzGr changed dashboard from one to another.

Reply:
FWIW I like ``ceph-mgr-core-modules``

Reply:
@smithfarm regarding to module versus plugin, i have a follow-up PR at #33435